projects/admin/src/app/circulation/checkin/checkin.component.spec.ts

```diff
@@ -23,13 +23,14 @@ import { TranslateModule } from '@ngx-translate/core';
 import { CoreModule, RecordModule } from '@rero/ng-core';
 import { CollapseModule } from 'ngx-bootstrap';
 import { UserService } from '../../service/user.service';
+import { SharedPipesModule } from '../../shared/shared-pipes.module';
 import { ItemComponent } from '../item/item.component';
 import { ItemsListComponent } from '../items-list/items-list.component';
 import { CardComponent } from '../patron/card/card.component';
 import { CheckinComponent } from './checkin.component';
 
 
-describe('CheckoutComponent', () => {
+describe('CheckinComponent', () => {
```

Comment:
This is not described in the commit message.

Reply:
done